packages/turf-isolines/index.d.ts

```diff
@@ -1,11 +1,13 @@
-/// <reference types="geojson" />
+import {Points, MultiLineStrings} from '@turf/helpers'
 
-type Points = GeoJSON.FeatureCollection<GeoJSON.Point>;
-type LineStrings = GeoJSON.FeatureCollection<GeoJSON.LineString>;
+interface Properties {
+    perIsoline: Array<Object>,
```

Comment:
👍 Looks good, only thing is since `perIsolines` & `toAllIsolines` are optional, you will need to add `:?` to make them optional.

```typescript
interface Properties {
    perIsoline?: Array<Object>,
    toAllIsolines?: Object
}
```

However, it might be best to not have those `properties` tags as an Object and just by their own param.

```typescript
declare function isolines(
    points: Points,
    breaks: number[],
    zProperty?:string,
    propertiesPerIsoline?: Object[],
    propertiesToAllIsolines?: Object): MultiLineStrings;
```

**javascript code**
```javascript
isolines(points, breaks, 'temperature', [{foo: 'bar'}], properties)
```